src/js/angular/rest/rdf4j.repositories.rest.service.js

```diff
@@ -84,4 +85,38 @@ function RDF4JRepositoriesRestService($http, $repositories) {
     function getGraphs(repositoryId) {
         return $http.get(`${REPOSITORIES_ENDPOINT}/${repositoryId}/contexts`);
     }
+
+
+    function resolveGraphs() {
+        let activeRepository = $repositories.getActiveRepository();
+        if (activeRepository !== "") {
+            return getGraphs(activeRepository).success(function (graphs) {
+                    graphs.results.bindings.unshift({
+                        contextID: {
+                            type: "default",
+                            value: "The default graph"
+                        }
+                    });
+
+                    Object.keys(graphs.results.bindings).forEach(function (key) {
+                        const binding = graphs.results.bindings[key];
+                        if (binding.contextID.type === "bnode") {
+                            binding.contextID.value = '_:' + binding.contextID.value;
+                        } else if (binding.contextID.type === "default") {
+                            binding.contextID.uri = "http://www.openrdf.org/schema/sesame#nil";
+                        } else {
+                            binding.contextID.uri = binding.contextID.value;
+                        }
+                    });
+                    graphs.results.bindings.unshift({
+                        contextID: {
+                            type: "all",
+                            value: "All graphs",
+                            uri: ""
+                        }
+                    });
+                    return  graphs.results.bindings;
```

Comment:
Are you sure this is correct? You now return just bindings whereas you expect graphs as i can see.